src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars__block">
+      <div class="stars--0">
+        <div class="star__item"></div>
```

Comment:
якщо по бем-у то stars__block, це елемент блоку stars, а такого блоку в тебе немає, тому по бем це непрвильно, краще зробити так 
```suggestion
    <div class="stars">
      <div class="stars--0">
        <div class="stars__item"></div>
```